### PR TITLE
adding droslean as a milestone maintainer

### PR DESCRIPTION
**Pull request by droslean**
/cc @kubernetes/release-team-leads 
/cc @bai 

Signed-off-by: Nikolaos Moraitis <nmoraiti@redhat.com>

### CODE REVIEW

**Review by hasheddan — APPROVED** [review]
/lgtm

**Review by savitharaghunathan — COMMENTED** [review]
/lgtm

File: config/kubernetes/sig-release/teams.yaml [hunk 1]
```diff
@@ -38,6 +38,7 @@ teams:
     - derekwaynecarr # Architecture / Node
     - dims # Architecture / Release
     - dougm # Release Manager
+    - droslean # Bug Triage Shadow 1.20
```
Comment (justaugustus) [inline config/kubernetes/sig-release/teams.yaml:41]:
```suggestion
    - droslean # 1.20 RT Bug Triage Shadow
```

**Review by justaugustus — CHANGES_REQUESTED** [review]
Nit: match the comment form of the other Release Team entries

File: config/kubernetes/sig-release/teams.yaml (hunk 1 above)
Comment (droslean) [reply config/kubernetes/sig-release/teams.yaml]: Thanks! Addressed.

**Pull request merged**